portal-ui/src/screens/Console/Configurations/utils.tsx

```diff
@@ -417,29 +417,45 @@ export const fieldsConfigurations: any = {
     },
     {
       name: "username_search_filter",
-      required: true,
+      required: false,
       label: "Username Search Filter",
       tooltip:
         'User search filter, for example "(cn=%s)" or "(sAMAccountName=%s)" or "(uid=%s)"',
       type: "string",
       placeholder: "Enter Username Search Filter",
     },
+    {
+      name: "user_dn_search_base_dn=",
+      required: false,
+      label: "User DN Search Base DN",
+      tooltip: "List of username search DNs, you can write one per field",
+      type: "csv",
+      placeholder: "Enter User DN Search Base DN",
+    },
+    {
+      name: "user_dn_search_filter=",
```

Comment:
```suggestion
      name: "user_dn_search_filter",
```

having `=` looks wrong